docs/topics/ref_openjdk-images-for-red-hat-enterprise-linux.adoc

```diff
@@ -0,0 +1,30 @@
+[id='openjdk-images-for-red-hat-enterprise-linux_{context}']
+= OpenJDK images for Red Hat Enterprise Linux
+You can select the OpenJDK image used to build and deploy your application to OpenShift. 
+OpenJDK images are available for RHEL{nbsp}7 and RHEL{nbsp}8 with OpenJDK 8 or OpenJDK 11.
+
+[cols="3*", options=header]
+|===
+|OS
+|JDK
+|Red Hat Container Catalog
+
+|RHEL 7
+|OpenJDK 8
+|link:https://access.redhat.com/containers/#/registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift[RHEL 7 with OpenJDK 8^]
+
+| 
+|OpenJDK 11
+|link:https://access.redhat.com/containers/#/registry.access.redhat.com/openjdk/openjdk-11-rhel7[RHEL 7 with OpenJDK 11^]
+
+|RHEL 8
+|OpenJDK 8
+|link:https://access.redhat.com/containers/#/registry.access.redhat.com/openjdk/openjdk-8-rhel8[RHEL 8 with OpenJDK 8^]
+
+| 
+|OpenJDK 11
+|link:https://access.redhat.com/containers/?tab=overview#/registry.access.redhat.com/openjdk/openjdk-11-rhel8[RHEL 8 with OpenJDK^]
+|===
+
+NOTE: The use of RHEL 8-based containers on a RHEL 7 host has limited support, which includes OpenShift 3 and OpenShift 4 installed on RHEL 7.
```

Comment:
For better clarity, how about rephrasing this sentence as:
The use of RHEL 8-based containers on a RHEL 7 host, for example, OpenShift 3 and OpenShift 4 installed on RHEL 7, has limited support.
For more information, see the link:https://access.redhat.com/articles/4087871[RHEL Container Compatibility Matrix].

Reply:
"OpenShift 3 or OpenShift 4 installed on a RHEL 7 host, is not fully supported." maybe? Just a suggestion :)

Reply:
Shorter is invariably better. As a colleague once told me, never use a short word when a diminutive one will do.

Reply:
Please check with PM about the support statements both in general and in context of the community version. Getting this right is imperative.

Reply:
@tradej Vert.x PM suggested the note. Will make sure he reviews the final version.